processes-webapp/src/main/webapp/vue-app/processes/components/RequestEditor.vue

```diff
@@ -67,7 +67,7 @@ export default {
   },
   watch: {
     inputVal: function (val) {
-      this.$emit('input', val);
+      this.$emit('input', val.replace('<p>', '').replace('</p>', ''));
```

Comment:
We should not just replace one HTML tag, there are other tags that could be present in the request.
The fix should just render correctly HTML on the notification content